generals/game.py

```diff
@@ -1,14 +1,18 @@
 import warnings
 import numpy as np
 import gymnasium as gym
-from typing import Dict, List
+from typing_extensions import TypeAlias
+
 from generals.config import DIRECTIONS, PASSABLE, MOUNTAIN, INCREMENT_RATE
 
 from scipy.ndimage import maximum_filter
 
+Observation: TypeAlias = dict[str, np.ndarray]
```

Comment:
This is not correct (probably was during your commit) -> now Observation is [a bit more complex](https://github.com/strakam/Generals-RL/blob/661858051d6fa39de801be85b412d8eed5d9a4a6/generals/game.py#L330), but idk how to 'type' it :(

Reply:
You are right, it was changed after my commit.